saw-remote-api/src/SAWServer/JVMCrucibleSetup.hs

```diff
@@ -96,7 +122,7 @@ interpretJVMSetup ::
   CryptolEnv ->
   [SetupStep JavaType] ->
   JVMSetupM ()
-interpretJVMSetup fileReader bic cenv0 ss = runStateT (traverse_ go ss) (mempty, cenv0) *> pure ()
+interpretJVMSetup fileReader bic cenv0 ss = runStateT (traverse_ go ss) (mempty, cenv0) $> ()
```

Comment:
If you use `evalStateT` instead of `runStateT` here, you won't need the `$> ()` part here, I believe.

Reply:
Changed, thanks!